buttercup.el

```diff
@@ -145,91 +157,91 @@ MATCHER is either a matcher defined with
 ;;; Built-in matchers
 
 (buttercup-define-matcher :not (obj matcher &rest args)
-  (let ((result (buttercup--apply-matcher matcher (cons obj args))))
-    (if (consp result)
-        (cons (not (car result))
-              (cdr result))
-      (not result))))
+                          (let ((result (buttercup--apply-matcher matcher (cons obj args))))
+                            (if (consp result)
+                                (cons (not (car result))
+                                      (cdr result))
+                              (not result))))
```

Comment:
Please load buttercup before re-indenting the file, buttercup defines correct indentation rules for `buttercup-define-matcher`.